Casks/7777.rb

```diff
@@ -0,0 +1,11 @@
+cask "7777" do
+  version "1.0.3"
+  sha256 "f74b2cb2ef792fc5ab81b5955a02d593dffe1972af61e20fca8a711ee017581b"
+
+  url "https://releases.port7777.com/1.0.3/macos/7777"
```

Comment:
How did you find this `url` without payment:

<img width="346" alt="Screen Shot 2020-11-19 at 09 58 16" src="https://user-images.githubusercontent.com/6127163/99610992-cd4a9e80-2a4d-11eb-939e-72b0763fe5e1.png">

According to their [GitHub support page](https://github.com/whilenull/7777-support#macos) the download link is simply `https://releases.port7777.com/latest/macos/7777`.

Reply:
Hi, 7777 author here. After looking at the documentation of Brew cask I would recommend:

```suggestion
  url "https://releases.port7777.com/#{version}/macos/7777"
```

That will simplify maintenance. I guess installing the `latest` version would work (wouldn't require a PR to update the `.rb` file for new versions), but isn't that contrary to the fact that we specify a specific version above?

> How did you find this url without payment:

The URL is not private, the app requires a license to run. So the process of installing the app via `brew cask` makes sense, you would then run `7777 configure` to setup the license.

Reply:
@mnapoli update can be triggered semi automated: https://github.com/Homebrew/homebrew-cask/blob/master/CONTRIBUTING.md

As far as I know, there is no 100% automated way, but I believe the cask maintainers are looking at it.